about/docinfo.xml

```diff
@@ -0,0 +1,12 @@
+<title>About</title>
+<productname>{product-title}</productname>
+<productnumber>{product-version}</productnumber>
+<subtitle>About OpenShift Service Mesh</subtitle>
```

Comment:
If OpenShift Service Mesh has an attribute in your docs, you could use it here.

Reply:
done